src/ziggurat/metrics.clj

```diff
@@ -17,37 +14,9 @@
              ^{:static true} [reportTime [String long] void]
              ^{:static true} [reportTime [String long java.util.Map] void]]))
 
-(defonce metrics-registry
-  (MetricRegistry.))
-
-(defn- merge-tags
-  [additional-tags]
-  (let [default-tags {"actor" (:app-name (ziggurat-config))}]
-    (merge default-tags (stringify-keys additional-tags))))
-
-(defn- get-tagged-metric
-  [metric-name tags]
-  (.tagged ^MetricName metric-name tags))
-
-(defn mk-meter
-  ([category metric]
-   (mk-meter category metric nil))
-  ([category metric additional-tags]
-   (let [namespace     (str category "." metric)
-         metric-name   (MetricRegistry/name ^String namespace nil)
-         tags          (merge-tags additional-tags)
-         tagged-metric (get-tagged-metric metric-name tags)]
-     (.meter ^MetricRegistry metrics-registry ^MetricName tagged-metric))))
-
-(defn mk-histogram
-  ([category metric]
-   (mk-histogram category metric nil))
-  ([category metric additional-tags]
-   (let [namespace     (str category "." metric)
-         metric-name   (MetricRegistry/name ^String namespace nil)
-         tags          (merge-tags additional-tags)
-         tagged-metric (.tagged ^MetricName metric-name tags)]
-     (.histogram ^MetricRegistry metrics-registry ^MetricName tagged-metric))))
+(defstate statsd-reporter
+  :start (metrics-lib/initialize (:datadog (ziggurat-config)))
```

Comment:
rebase?

Reply:
yes rebased it.

Reply:
oh I mean get the config thru `(config/statsd-config)` because this behavior was changed on the previous commit

Reply:
will dropwizard be the default metrics implementation?

Reply:
oh, sure, will fix that